app/controllers/editions_controller.rb

```diff
@@ -93,7 +93,7 @@ def update
   end
 
   def review
-    if resource.reviewer
+    if resource.reviewer and resource.reviewer.present?
```

Comment:
You don't need the `and` clause here, this line could just be `if resource.reviewer.present?`.